src/Symfony/Component/Process/Tests/ProcessTest.php

```diff
@@ -1342,6 +1342,21 @@ public function testNonBlockingNorClearingIteratorOutput()
         $this->assertSame($expectedOutput, $output);
     }
 
+    public function testChainedProcesses()
+    {
+        $p1 = new Process(self::$phpBin.' -r '.escapeshellarg('fwrite(STDERR, 123); fwrite(STDOUT, 456);'));
```

Comment:
We can hav only one getProces per test case. And there is not need to configure differently the process for sigchild. So no need.